force_torque_sensor_broadcaster/src/force_torque_sensor_broadcaster.cpp

```diff
@@ -87,12 +90,43 @@ controller_interface::CallbackReturn ForceTorqueSensorBroadcaster::on_configure(
         torque_names.z));
   }
 
+  try
+  {
+    filter_chain_ =
+      std::make_unique<filters::FilterChain<WrenchMsgType>>("geometry_msgs::msg::WrenchStamped");
+  }
+  catch (const std::exception & e)
+  {
+    fprintf(stderr, "Exception thrown during filter chain creation with message : %s \n", e.what());
+    return CallbackReturn::ERROR;
+  }
+
+  // As the sensor_filter_chain parameter is of type 'none', we cannot directly check if it is
+  // present. Even if the sensor_filter_chain parameter is not specified, the filter chain will be
+  // correctly configured with an empty list of filters (https://github.com/ros/filters/issues/89).
```

Comment:
```suggestion
  // correctly configured with an empty list of filters.
```
The issue is kind of resolved https://github.com/ros/filters/issues/89?